mlir/include/mlir/Dialect/NVGPU/IR/NVGPU.td

```diff
@@ -192,6 +192,15 @@ def NVGPU_WarpgroupMatrixDescriptor : NVGPU_Type<"WarpgroupMatrixDescriptor", "w
   let assemblyFormat = "`<` struct(params) `>`";
 }
 
+def NVGPU_WarpgroupResult : NVGPU_Type<"WarpgroupResult", "warpgroup.result", []> {
+  let parameters = (ins "Type":$tensor);
+  let assemblyFormat = "`<` struct(params) `>`";
+  let description = [{
+    It is fragmented result matrix from `nvgpu.warpgroup.mma`.
```

Comment:
What is "it" in this context?
Maybe something more descriptive:
"This type represents a partial result of the expansion of `nvgpu.warpgroup.mma`?"

Putting an example would be great: Pattern IN -> produced pattern.